root/treeformula/event/dt_wrap.C

```diff
@@ -1,5 +1,6 @@
+#include "dt_RunDrawTest.C"
+
 void dt_wrap(const char* from, Int_t mode = 0, Int_t verboseLevel = 0) {
-   gROOT->ProcessLine(".L dt_RunDrawTest.C+");
```

Comment:
That's a change of functionality - if possible please leave the original version, and add the missing `#include`s to `dt_RunDrawTest.C`

Reply:
@pcanal remind me - why do we need to pre-build the ACLiC library through GNUMake instead of "just" letting ACLiC do its job?

Reply:
This has 2 advantages
* If the build fails, it stops at its own stage
* Reduce the log pollution in case of any warning or compiler messages.

Reply:
> If the build fails, it stops at its own stage

Agreed (at the cost of less coverage of ACLiC). OTOH when switching to CMake, having test-prerequisites built as a separate step might be more of a pain than a gain: as you know, `ctest foo` doesn't (necessarily) build foo's dependencies...

> Reduce the log pollution in case of any warning or compiler messages.

This one I find problematic. We are not seeing many of the warnings in our test code, sometimes making failure analysis more difficult.

But let's discuss by voice - maybe tomorrow. Thanks for reminding me of the rationale for the current setup!